src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="home-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="home-page__content">
+
+    <div class="card" data-qa="card">
+
+      <img
+          src="./images/imac.jpeg"
+          alt="card__image"
+          class="card__image">
+
+      <div class="card__info">
+        <h2 class="card__info-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__info-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__info-reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+            Reviews: 5
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-title">
+            Price:
+          </div>
+          <div class="card__price-value">
+            $2,199
+          </div>
```

Comment:
The same thing I think it's better to use paragraph instead of div

Reply:
Not sure, because i think <p> has means that it's paragraph, and must starts from new line

Reply:
> Not sure, because i think p has means that it's paragraph, and must starts from new line

Yeah, but you can use flex to place them on a same line